components/dialog/dialog-fullscreen.js

```diff
@@ -251,7 +250,6 @@ class DialogFullscreen extends LocalizeCoreElement(AsyncContainerMixin(DialogMix
 			<div style=${styleMap(slotStyles)}><slot></slot></div>
 		`;
 
-		if (!this._titleId) this._titleId = getUniqueId();
```

Comment:
I think we should just set `this._titleId = getUniqueId()` in the constructor. We do this already in a bunch of places, and it would avoid needing to do it in `willUpdate`.